src/main/java/org/jabref/logic/net/URLDownload.java

```diff
@@ -132,6 +133,20 @@ public void checkServerTrusted(X509Certificate[] chain, String authType) {
         }
     }
 
+    /**
+     *
+     * @param sf trust manager
+     * @param v host verifier
+     */
+    public static void setSSLVerification(SSLSocketFactory sf, HostnameVerifier v) {
```

Comment:
```suggestion
    public static void setSSLVerification(SSLSocketFactory socketFactory, HostnameVerifier verifier) {
```
Please no single character abbreviations as variable names